alfasim_sdk/_alfasim_sdk_utils.py

```diff
@@ -2,7 +2,7 @@
 
 import attr
 
-from alfasim_sdk.types import BaseField, Tab, Tabs
+from alfasim_sdk.types import BaseField, Group, Tab, Tabs
 
 
 def get_attr_class(class_: type, caption: str, icon: Optional[str], model: Optional[type], bases: Optional[Tuple]=()):
```

Comment:
`Optional` type hint is for argument that accept `None` value and not those optional due having a default value.
Maybe it has a non `None` default but will accept `None` then this is just a comment =)

Reply:
I once read on mypy cheat sheet that is a good practice to provide type annotation to variables that initialized with "None" =)

I know that the instructions clearly say variables, but I think that it's a good argument to use it on  function arguments :stuck_out_tongue_closed_eyes: 
[Link for the cheat_ sheet](https://mypy.readthedocs.io/en/latest/cheat_sheet_py3.html#when-you-re-puzzled-or-when-things-are-complicated)